src/cbc_sdk/platform/alerts.py

```diff
@@ -1603,6 +1631,25 @@ def set_group_by(self, field):
         self._group_by = field
         return self
 
+    def get_alert_search_query(self):
+        """
+        Converts the GroupedAlertSearchQuery into a nongrouped AlertSearchQuery
+
+        Returns: AlertSearchQuery
+
+        Note: Does not preserve sort criterion
+        """
+        ignored_keys = ["_doc_class", "_cb", "_count_valid", "_total_results", "_query_builder", "_sortcriteria"]
+        alert_search_query = self._cb.select(Alert)
+        for key, value in vars(alert_search_query).items():
+            if hasattr(self, key) and key not in ignored_keys:
+                setattr(alert_search_query, key, self.__getattribute__(key))
+        key = "_time_range"
+        if hasattr(self, key):
+            setattr(alert_search_query, key, self.__getattribute__(key))
```

Comment:
I'd avoid making a temporary variable here as it leads to a bit of confusion with the for loop above. Also is there a reason the for loop would not catch this it seems redundant

Reply:
Same applies for set_group_by

Reply:
As discussed in the meeting, we should set a default `self._time_range = {}` in order to reduce unique logic for individual properties